src/Chapter2.hs

```diff
@@ -740,7 +757,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n xs = filter (elem n) xs
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function as well  🙂 
But your implementation is already great 💯

Reply:
I was able to use point free style with this.  Does it suffer in readability? Is that considered "fancy Haskell"?

Reply:
I think in this particular case it is still readable.

Haha, "fancy Haskell" is a bit of the ambiguous term, and every one has its own understanding of it 😄 Many people do like point free, so many are against it. I think it is okay and sometimes beneficial, but you are right, that there should be some common sense of when you have to stop eta-reducing 😄